app/actors/hyrax/actors/file_set_actor.rb

```diff
@@ -143,36 +157,10 @@ def label_for(file)
           end
         end
 
-        # Takes an optional block and executes the block if the save was successful.
-        # @return [Boolean] false if the save was unsuccessful
-        def save
```

Comment:
Can you also remove the dependency on `retriable` from the gemspec? This was the only code that used that gem, AFAIK, so now that it's gone we no longer need the dependency.